lib/src/constants.dart

```diff
@@ -1,3 +1,13 @@
+/// URL endpoint for sending Google Analytics Events
+const String kAnalyticsUrl = 'https://www.google-analytics.com/mp/collect';
+
+/// Name for the text file that will contain the
+/// user's randomly generated client id
+const String kClientIdFileName = 'CLIENT_ID';
+
+/// Name for the file where telemetry status and tools data will be stored
+const String kConfigFileName = 'dart-flutter-telemetry.config';
```

Comment:
can this be `dash-analytics.config`, so it's obvious to someone doing `ls .dart-tool` what it is?